conans/test/unittests/client/generators/cmake_test.py

```diff
@@ -557,37 +557,49 @@ def setUp(self):
         cpp_info = CppInfo(ref.name, "dummy_root_folder1")
         cpp_info.filter_empty = False  # For testing purposes only
         cpp_info.name = ref.name
-        cpp_info.build_modules = ["my-module.cmake"]
```

Comment:
Will the assignment fail? I see the existing implementation keeps ``append()`` and ``extend()``, but not sure about the ``build_modules = ["..."]`` one. Would this be breaking?

Reply:
No, you have a specific test for that case here https://github.com/conan-io/conan/pull/8232/files#diff-3779db7bbc92daafae06a9e61c096a7970fe9fb3e9345183947061183d7c5612R249